client/src/components/Home/Filters.js

```diff
@@ -0,0 +1,30 @@
+//ToDo: Add label to the left side of the filters
+//ToDo: Add search icon button to search with filters --> Collect filter and only if the user presses search apply filter
+//ToDo: Make it possible to clear a filter
```

Comment:
All of this was the goal for this sprint?

Reply:
Did not motice the todos, sorry. Will address the design-related requests till  27.06